var/spack/repos/builtin/packages/openmp-extras/package.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+tools_url = 'https://github.com/ROCm-Developer-Tools'
+compute_url = 'https://github.com/RadeonOpenCompute'
+
+# 3.9 SHA Keys
+openmp_extras39 = dict()
+openmp_extras39 = {
+    "aomp":
+        "377ab59b685a73b3f95fba95f5e028678ec5aafabc4177b7f0ffb78da095d679",
+    "devlib":
+        "c99f45dacf5967aef9a31e3731011b9c142446d4a12bac69774998976f2576d7",
+    "llvm":
+        "1ff14b56d10c2c44d36c3c412b190d3d8cd1bb12cfc7cd58af004c16fd9987d1",
+    "flang":
+        "5d113f44fb173bd0d5704b282c5cebbb2aa642c7c29f188764bfa1daa58374c9",
+    "extras":
+        "830a37cf1c6700f81fc00749206a37e7cda4d2867bbdf489e9e2d81f52d06b3d"
+
+}
+
+
+class OpenmpExtras(Package):
+    """OpenMP support for ROCm LLVM."""
```

Comment:
The name of the package feels a bit too generic...

Reply:
I can understand that. We could possibly change the package name to AmdgpuOpenmpExtras or RocmOpenmpExtras.

Reply:
Changed to rocm-openmp-extras.